source/Log4Tc/Mbc.Log4Tc.Output.NLog/NLogLog4TcOutputPlugin.cs

```diff
@@ -9,6 +9,10 @@ public class NLogLog4TcOutputPlugin : IPlugin
         public void ConfigureServices(IServiceCollection services, IConfiguration configuration)
         {
             services.AddSingleton<IOutputFactory, NLogLog4TcOutputFactory>();
+
+            // Example see: https://blog.skrots.com/nlog-implementation-with-azure-application-insights/?feed_id=1833&_unique_id=64f62b11bebcc
+            // See Application Insights logging adapters: https://github.com/microsoft/ApplicationInsights-dotnet/blob/c420e04562876791f27a61644760d7b7512832d9/LOGGING/README.md#nlog
+            services.AddApplicationInsightsTelemetryWorkerService();
```

Comment:
Wäre es nicht sinnvoll, die nur dann hinzuzufügen, wenn sie wirklich benutzt werden?

Reply:
Der Aufruf bewerkstelligt die die Verfügbarkeit in nlog. Aktiviert wird es dann über die Konfig im nlog.config als target.

Alternative ist das setzen eines Konfig parameters. Man gewinnt wie nichts damit bzw verliert nix wenn man es immer lädt.